src/platforms/stlink/README.md

```diff
@@ -45,10 +45,10 @@ NB: SWDIO/TMS is on P**B**14, not P**A**14.
 ## Upload BMP Firmware
 
 * Keep the original ST Bootloader.
-* Compile firmware with `make PROBE_HOST=stlink ST_BOOTLOADER=1`
+* Compile firmware with the option `-Dbmd_bootloader=false`
 * Upload firmware with stlink-tool from [stlink-tool](https://github.com/UweBonnes/stlink-tool/tree/stlinkv21)(*3).
```

Comment:
This link needs updating and switching to https://github.com/blackmagic-debug/stlink-tool

Reply:
Done.